### PR TITLE
Accent Change For Character Creation Page + Irish Accent 

### PR DESCRIPTION
**Pull request by SsalamethVersaach**
## About the PR

This PR changed accents weights and also adds the ability to choose more accents in the character menu, as well as a new Irish accent. I did some yaml changes, along with adding a new accent.

## Why / Balance

The reason behind this change was due to my own desire for people to have more accent choices, and to allow people to choose French, Russian, Spanish, Irish, and Mobster on round start. Part of this also stemmed from my desire to make it easier for players to diversify their characters.

## Technical details

I changed the cost of some speech attributes in the speech.yml file, as well as added additional information and descriptions for these accents within the ftl files. In addition, I created an Irish accent which utilizes the irish.ftl and word_replacements.yml files- and added an accent system for it to function. 

## Media

screenshine

image

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

No breaking changes (that I know of).

**Changelog**

:cl:
- add: Added more accents on the character creation page, and the brand new Irish accent!

### CODE REVIEW

File: RobustToolbox
Comment (DangerRevolution) [inline RobustToolbox:1]: Bad

File: Resources/Prototypes/Traits/speech.yml
Comment (DangerRevolution) [inline Resources/Prototypes/Traits/speech.yml:1]: This needs to be in a Delta-V folder

File: Resources/Prototypes/DeltaV/Traits/neutral.yml
Comment (DangerRevolution) [inline Resources/Prototypes/DeltaV/Traits/neutral.yml:1]: Why?

File: Resources/Prototypes/Traits/speech.yml [hunk 1]
```diff
@@ -64,14 +64,12 @@
...
 
-# 2 Cost
-
 - type: trait
   id: SocialAnxiety
   name: trait-socialanxiety-name
   description: trait-socialanxiety-desc
   category: SpeechTraits
-  cost: 2
+  cost: 1
```
Comment (DangerRevolution) [inline Resources/Prototypes/Traits/speech.yml]: This is unrelated to your PR, and should be a commented change

File: Resources/Prototypes/Traits/speech.yml [hunk 2]
```diff
@@ -84,6 +82,51 @@
   name: trait-frontal-lisp-name
   description: trait-frontal-lisp-desc
   category: SpeechTraits
-  cost: 2
+  cost: 1
```
Comment (DangerRevolution) [inline Resources/Prototypes/Traits/speech.yml]: This is unrelated to your PR, and should be a commented change

File: Resources/Locale/en-US/traits/traits.ftl
Comment (DangerRevolution) [inline Resources/Locale/en-US/traits/traits.ftl:1]: This needs to be in a Delta-V folder.

Russian accent is really bad and leads to extremely poor readability. Should not be used.

The Descriptions are also just copy and pasted with minor edits; should be unique.

**Review by DangerRevolution — CHANGES_REQUESTED** [review]
Russian accent is up for debate but stuff needs to be commented / in delta-V folders

**Review by SsalamethVersaach — COMMENTED** [review]
I think I fixed everything (hopefully).

Please let me know what else is needed. 

Thanks.

File: Resources/Prototypes/DeltaV/Traits/neutral.yml
Comment (SsalamethVersaach) [reply Resources/Prototypes/DeltaV/Traits/neutral.yml:1]: This was where I was going to put the new traits, just was testing there prior and forgot about it.

File: Resources/Prototypes/Traits/speech.yml (hunk 1 above)
Comment (SsalamethVersaach) [reply Resources/Prototypes/Traits/speech.yml]: It was related due to me wanting more variety, but comments have been added-

File: Resources/Prototypes/Traits/speech.yml (hunk 2 above)
Comment (SsalamethVersaach) [reply Resources/Prototypes/Traits/speech.yml]: It was related due to me wanting more variety, but comments have been added (to all of them)

File: Resources/Locale/en-US/traits/traits.ftl
Comment (SsalamethVersaach) [reply Resources/Locale/en-US/traits/traits.ftl:1]: Russian Accent was removed, traits were moved to a Delta-V folder, and I changed descriptions.

File: RobustToolbox
Comment (SsalamethVersaach) [reply RobustToolbox:1]: I don't know what to do about this as it's a submodule and automatically populated in the PR.

**Comment by SsalamethVersaach** [comment]
Apparently GitHub has decided to PR literally EVERYTHING but my changes in this so I am just going to re-PR it, I give up on comprehending it. Apologies.
